FreeRTOS/Demo/RISC-V-Qemu-virt_GCC/Makefile

```diff
@@ -0,0 +1,53 @@
+CROSS=riscv64-unknown-elf-
+CC=$(CROSS)gcc
+OBJCOPY=$(CROSS)objcopy
+ARCH=$(CROSS)ar
+
+RTOS_SOURCE_DIR=../../Source
+DEMO_SOURCE_DIR=../Common/Minimal
+LIBWRAP_SOURCE_DIR=./freedom-e-sdk/libwrap
+
+CPPFLAGS = \
+	-g -O2 -Wall \
```

Comment:
could you please add a debug flag (DEBUG=1) to pass to the makefile to add debugging symbols with   (-Og -ggdb3)